src/app/meta-campaign/meta-campaign.component.ts

```diff
@@ -394,7 +394,7 @@ export class MetaCampaignComponent implements AfterViewChecked, OnDestroy, OnIni
       // use a too-low Y position when lots of cards were shown and we didn't have a delay, both with `scrollToAnchor()`
       // and manual calculation + `scrollToPosition()`.
 
-      this.autoScrollTimer = setTimeout(() => {
+      this.autoScrollTimer = window.setTimeout(() => {
```

Comment:
using `window` tells TS that we're only interested in the function from the Browser api, not the one from node that returns a different type.